src/i2c.rs

```diff
@@ -19,6 +19,8 @@ use crate::hal::blocking::i2c::{Read, Write, WriteRead};
 
 #[cfg(feature = "stm32l0x2")]
 use crate::dma;
+#[cfg(feature = "stm32l0x2")]
+use crate::dma::Buffer;
```

Comment:
Suggestions: Extend the `use` statement above instead, then you can save one `#[cfg(...)]`. Something like this:
``` rust
#[cfg(feature = "stm32l0x2")]
use crate::dma::{
    self,
    Buffer,
};
```